bin/elasticsearch-reindex.js

```diff
@@ -27,6 +27,7 @@ cli
 .option('-n, --max_docs [value]', 'default -1 unlimited', -1)
 .option('-v, --api_ver [value]', 'default 1.5', '1.5')
 .option('-p, --parent [value]', 'if set, uses this field as parent field', '')
+.option('-m, --promise [value]', 'if set indexes expecting promises, default: false', false)
```

Comment:
I'm not very familiar with commander. This line could be improved so that just by passing `-m` it would change the value to `true`. The only part I don't like now is that you need to explicitly specify `-m true …`